models/user.js

```diff
@@ -0,0 +1,75 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+const crypto = require('crypto');
+const jwt = require('jsonwebtoken');
+
+var userSchema = new mongoose.Schema({
+  username: { type: String, unique: true, required: true },
+  password: { type: String, required: true },
+  findHash: { type: String, unique: true },
```

Comment:
Let's add in two security questions to the schema, in case we decide to implement the password reset feature. I'd think they would be:
secQuestion1 { type: String, unique: true },
secQuestion2 { type: String, unique: true },
secAnswer1: String,
secAnswer2: String,